src/README.md

```diff
@@ -2,13 +2,17 @@
 
 [![CI](https://github.com/grafana/grafana-advisor-app/actions/workflows/push.yaml/badge.svg)](https://github.com/grafana/grafana-advisor-app/actions/workflows/push.yaml)&nbsp;&nbsp;![Latest Version Badge](https://img.shields.io/badge/dynamic/json?logo=grafana&query=$.version&url=https://grafana.com/api/plugins/grafana-advisor-app&label=Version&prefix=v&color=F47A20)
 
-An app for visualising checks that require immediate action or investigation.
+> [!NOTE]
+> Grafana Advisor is currently [experimental](https://grafana.com/docs/release-life-cycle/#quick-reference-table)
+
+A centralized view to help Grafana administrators keep their instances running smoothly. It includes a suite of  checks which flag issues requiring action or further investigation.
 
 <img width="500px" src="https://raw.githubusercontent.com/grafana/grafana-advisor-app/main/docs/screenshot.png" />
 
 ## Requirements
 
-It is also necessary to enable the `grafanaAdvisor` [feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) in the Grafana server.
+- Grafana v12.0.0
```

Comment:
we had this before and I removed it because it became stale, since we already have the Grafana dependency section, I think we can omit this?
<img width="1010" alt="Screenshot 2025-04-16 at 15 02 05" src="https://github.com/user-attachments/assets/75a855d4-59af-4d62-a704-42e2b78a5d97" />

Reply:
okay